src/main/java/lt/vu/persistence/orm/entities/Order.java

```diff
@@ -18,7 +18,7 @@
     private int id;
 
     @Column(name = "CREATED_AT", nullable = false)
-    private Date createdAt;
+    private Date createdAt = new Date();
```

Comment:
I think `createdAt` could be current timestamp.

Reply:
Correct, this could even be readonly